SwiftUITask/SwiftUITask/Preview Content/ContentView.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  ContentView.swift
+//  SwiftUITask
+//
+//  Created by Алексей Барлетов on 30.04.2024.
+//
+
+import SwiftUI
+///Главный экран приложения
+struct ContentView: View {
+    enum Constant {
+        static let shareImage = "share"
+        static let directionImage = "direction"
+        static let messageTextActionSheet = "Сохранен в папку загрузки"
+        static let textButtonActionSheet = "OK"
+        static let textButtonAlert = "Поделиться?"
+        static let oKbutton = "Да"
+        static let nObutton = "Нет"
+        static let leftButtonImage = "left"
+        static let rideButtonImage = "ride"
+    }
+    
+    @State private var progress: Float = 0
+    @State private var buttonPlay = false
+    @State private var isPlaying = false
+    @State private var isPropertyShare = false
+    @State private var isPropertyDirection = false
```

Comment:
```suggestion
    @State private var isPropertyDirection = false
    добавить пробел между стейт свойствами и обсерв
```
<img width="557" alt="Снимок экрана 2024-05-03 в 14 37 22" src="https://github.com/AlexeyBarletov/SwiftUI/assets/109034433/9d8fe766-a4e5-4616-8975-ec7b5fc505ff">